### ISSUE TITLE
NABU file

### ISSUE THREAD

**Issue by cavaughankirov** [lesbird/MSX8#1]
Tried several times downloading the NABU zip file, but it is consistently unable to be unarchived. Something's wrong with it.

**Comment by lesbird** [comment]
Are you still having issues with this? I just downloaded and it is unzipping for me so seems to be ok. MSX8NABU.ZIP

**Comment by cavaughankirov** [comment]
Yeah. Someone else thought I must be downloading it wrong, but I don't see how that's possible. I've tried right clicking on MSX8NABU.zip and saving it and clicking on it, going to the RAW option and downloading it. Regardless, if I try to unzip it, I get:

> unzip MSX8NABU.zip
Archive:  MSX8NABU.zip
  End-of-central-directory signature not found.  Either this file is not
  a zipfile, or it constitutes one disk of a multi-part archive.  In the
  latter case the central directory and zipfile comment will be found on
  the last disk(s) of this archive.
unzip:  cannot find zipfile directory in one of MSX8NABU.zip or
        MSX8NABU.zip.zip, and cannot find MSX8NABU.zip.ZIP, period.

If I just try unarchiving it from File Viewer it also errors. 

I was able to get a copy of it from that other person though.
